packages/twenty-website/src/content/developers/self-hosting/troubleshooting.mdx

```diff
@@ -51,10 +51,25 @@ This should work out of the box with the eslint extension installed. If this doe
 
 In `packages/twenty-front/.env` uncomment `VITE_DISABLE_TYPESCRIPT_CHECKER=true` and `VITE_DISABLE_ESLINT_CHECKER=true` to disable background checks thus reducing amount of needed RAM.
 
+**If it does not work:**  
+Another thing you can try is to only run the services you need instead of `npx nx start`. For instance, if you work on the server, try to run only `npx nx worker twenty-server`
+
+**If it does not work:**  
+Well, you will need to investigate which process are taking you so much RAM on your machine. At twenty, we noticed that some VScode extensions were taking a lot of RAM so we temporarily disable them. 
```

Comment:
It should be `Twenty`, not `twenty`, after all, that's the name of project/product